content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx

```diff
@@ -2,14 +2,37 @@ import {connect} from "react-redux";
 import React from "react";
 
 export class _Hero extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    const feed = this.props.DiscoveryStream.feeds[this.props.feed.url];
+
+    this.state = {
+      recommendations: feed.data.recommendations
```

Comment:
I dunno what happens if you put something stored in redux into a component's state. Example, if the redux store changes, I don't know what happens to the data in the component's state. Might work, dunno.

Reply:
What's the alternative?

Reply:
Typically just read the data from `props`